src/styles/utils/_reset.scss

```diff
@@ -0,0 +1,27 @@
+textarea {
```

Comment:
I would recommend adding reset to the html with the cdn link instead of the file
CDN will cache and it will optimize page rendering